### PR TITLE
Fix bug in BatchRenormalization

### PR DESCRIPTION
**Pull request by DaniyarM**
Checking if broadcasting need in same way as in `BatchNormalization` layer. Otherwise it caused an error with `Keras 2.2.4`

### CODE REVIEW

**Comment by rgreenblatt** [comment]
I think this change is the same as the change I made here https://github.com/keras-team/keras-contrib/pull/325. I didn't realize this addressed the same problem at first.

**Comment by gabrieldemarmiesse** [comment]
Closing as #325 was merged.
